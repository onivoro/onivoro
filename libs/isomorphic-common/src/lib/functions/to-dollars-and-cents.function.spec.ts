import { toDollarsAndCents } from './to-dollars-and-cents.function';

describe('toDollarsAndCents', () => {
    it('worx', () => {
        expect(toDollarsAndCents(337.33)).toMatchSnapshot();
    });
});
