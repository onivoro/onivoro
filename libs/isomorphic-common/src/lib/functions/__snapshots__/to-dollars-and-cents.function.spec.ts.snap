// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toDollarsAndCents worx 1`] = `"three hundred thirty-seven dollars and thirty-three cents"`;
